admiral/pkg/clusters/serviceentry.go

```diff
@@ -24,6 +25,13 @@ import (
 	"github.com/istio-ecosystem/admiral/admiral/pkg/controller/util"
 )
 
+type SeDrTuple struct {
+	SeName string
```

Comment:
do the SeName and DrName need to be separate fields in the struct? Since they will also be accessible via ServiceEntry.Name, etc.

Reply:
ServiceEntry and DestinationRule in this struct are istio model objects and not k8s objects, so the `Name` field isn't available.